api/tests/Api/ContentNodes/ContentNode/ReadContentNodeTest.php

```diff
@@ -52,20 +51,4 @@ public function testGetSingleContentNodeIsAllowedInCampPrototype() {
         // then
         $this->assertResponseStatusCodeSame(200);
     }
-
-    public function testGetSingleContentNodeIncludesProperRelationLinks() {
```

Comment:
Müsste man hier einen Test als Ersatz schreiben, der überprüft dass der JSON-Inhalt rauskommt?

Reply:
Tests extended in https://github.com/ecamp/ecamp3/pull/2825/commits/dd67ee301305c9596c196ba0562d973d13fb0c69